lib/cipher.rb

```diff
@@ -8,7 +8,7 @@ def initialize
   end
 
   def generate_key
-    rand(99999).to_s.rjust(5, '0')
+    "0" + rand(9999).to_s.rjust(4, "0")
```

Comment:
This method doesn't look _as_ sleek as it did, but it wasn't generating keys with leading zeros before the change.